sfaira/ui/user_interface.py

```diff
@@ -59,16 +59,20 @@ def __init__(
         self.use_sfaira_repo = sfaira_repo
         self.cache_path = os.path.join(cache_path, '')
         self.adata_ids = AdataIdsSfaira()
+        self.sfaira_repo_url = "https://zenodo.org/record/4836517/files/"
 
         if sfaira_repo:  # check if public sfaira repository should be accessed
-            self.model_lookuptable = self._load_lookuptable("https://zenodo.org/record/4836517/files/")
```

Comment:
This should probably be in `sfaira/consts/`.

Reply:
done